src/substrate-indexed-data/orders/order.service.ts

```diff
@@ -19,10 +21,11 @@ export class OrderService {
           }
         }
       })
-
-      const hits_order = order.body.hits.hits;
-
-      return hits_order.length > 0 ? hits_order[0]._source : null;
+      hits_order = order.body.hits.hits
+    } catch (error) {
+      console.log('API "orders/:hash_id":', error.body.error.reason);
```

Comment:
Instead of `console.log` use Loggers instead.

https://docs.nestjs.com/techniques/logger

Reply:
Either throw a new exception or return an `HttpException`. Don't return default values when error.